<launch>
  <!-- <arg name="robot_name"        default="robot2"/> //-->
  <arg name="synchroManage/debugType" default="1"/>
  <!-- <group ns="$(arg robot_name)"> -->
 <!-- <node pkg="autonomous_mobile_robot" name="autonomous_mobile_robot_syncro" type="autonomous_mobile_robot_syncro" output="screen"/> -->
 <node pkg="autonomous_mobile_robot_2022" name="autonomous_mobile_robot_2022_si" type="autonomous_mobile_robot_2022_si">
  <param name="debugType" value="$(arg synchroManage/debugType)"/>
 </node> 
 <node pkg="autonomous_mobile_robot_2022" name="autonomous_mobile_robot_2022_frm" type="autonomous_mobile_robot_2022_frm">
  <param name="frameRate" value="15"/>
 </node>
  <!-- </group> -->
</launch>
